brave-kafka/src/main/java/com/github/kristofa/brave/SpanUtils.java

```diff
@@ -0,0 +1,62 @@
+
+/**
+ * Copyright 2016-2017 SmartThings
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.github.kristofa.brave;
+
+import brave.propagation.TraceContext;
+
+/**
+ * Contains some util functions to get around some access modifiers
+ */
+public class SpanUtils {
```

Comment:
SpanId is part of the now deprecated Brave 3.x APIs.  Just pointing this out as something we need to cleanup.  That can happen in another PR such as https://github.com/SmartThingsOSS/smartthings-brave/pull/8